Module not found: Error: Cannot resolve 'file' or 'directory' for webpack-dev-server/client

**Bug**
The webpack-dev-server client module is being incorrectly resolved.

**Steps to reproduce**

Run `dojo build -w`

**Expected behavior:**

To not throw an error.

**Actual behavior:**
`Webpack Module not found: Error: Cannot resolve 'file' or 'directory' for webpack-dev-server/client`

--- a/src/main.ts
+++ b/src/main.ts
@@ -1,6 +1,5 @@
 import { Command, Helper } from 'dojo-cli/interfaces';
 import { Argv } from 'yargs';
-import { join } from 'path';
 const webpack: any = require('webpack');
 const WebpackDevServer: any = require('webpack-dev-server');
 const config: any = require('./webpack.config');
@@ -20,7 +19,7 @@ interface WebpackOptions {
 
 function watch(config: any, options: WebpackOptions, args: BuildArgs): Promise<any> {
 	config.devtool = 'eval-source-map';
-	config.entry.unshift(join(__dirname, 'node_modules', 'webpack-dev-server/client?'));
+	config.entry.unshift('webpack-dev-server/client?');
 
 	const compiler = webpack(config);
 	const server = new WebpackDevServer(compiler, options);

--- a/tests/unit/main.ts
+++ b/tests/unit/main.ts
@@ -1,4 +1,3 @@
-import { join } from 'path';
 import { beforeEach, afterEach, describe, it } from 'intern!bdd';
 import * as assert from 'intern/chai!assert';
 import MockModule from '../support/MockModule';
@@ -72,7 +71,7 @@ describe('main', () => {
 			assert.equal(mockWebpackConfig.devtool, 'eval-source-map');
 			assert.deepEqual(
 				mockWebpackConfig.entry,
-				[join(require.toUrl('src'), 'node_modules', 'webpack-dev-server/client?')]
+				['webpack-dev-server/client?']
 			);
 		});
 	});

